apis/management.cattle.io/v3/multi_cluster_app.go

```diff
@@ -39,3 +46,23 @@ type Answer struct {
 	ClusterName string            `json:"clusterName,omitempty" norman:"type=reference[cluster]"`
 	Values      map[string]string `json:"values,omitempty" norman:"required"`
 }
+
+type MultiClusterAppRevision struct {
```

Comment:
lets make sure to sync with @mrajashree on the rbac. The same rbac rules should apply to the revision that apply to the app